debian/libibumad3.install

```diff
@@ -1 +1,2 @@
 usr/lib/*/libibumad*.so.*
+lib/udev/rules.d/libibumad.rules
```

Comment:
@bdrung @jarodwilson is having a  udev .rule in a library package OK for packaging, multi-arch/etc ?